pkg/controller/build/build_controller_test.go

```diff
@@ -124,6 +124,29 @@ var _ = Describe("Reconcile Build", func() {
 				Expect(statusWriter.UpdateCallCount()).To(Equal(1))
 				Expect(reconcile.Result{}).To(Equal(result))
 			})
+			It("fails when no any secret exists in namespace", func() {
+				// Fake some client LIST calls and ensure we populate all
+				// different resources we could get during reconciliation
+				client.ListCalls(func(context context.Context, object runtime.Object, _ ...crc.ListOption) error {
+					switch object := object.(type) {
+					case *corev1.SecretList:
+						list := ctl.FakeNoSecretListInNamespace()
+						list.DeepCopyInto(object)
+					case *build.ClusterBuildStrategyList:
+						list := ctl.ClusterBuildStrategyList(buildStrategyName)
+						list.DeepCopyInto(object)
+					}
+					return nil
+				})
+				statusCall := ctl.StubFunc(corev1.ConditionFalse, fmt.Sprintf("there are no secrets in namespace %s", namespace))
```

Comment:
`there are no secrets in namespace` -> `there is no secret in namespace` ?

Also same for line 147

Reply:
Wording-wise, I think both is fine @zhangtbj.

Reply:
@zhangtbj @SaschaSchwarze0 Thanks for your review! These statements are same with [build_controller.go](https://github.com/shipwright-io/build/blob/master/pkg/controller/build/build_controller.go). If I change them, then I also need to change all statements in build_controller.go. Should I do?

Reply:
aha, I am not good at the wording.

If both are fine, I am ok to keep this.